products/tests.py

```diff
@@ -1,3 +1,289 @@
+import json
+
 from django.test import TestCase
+from django.test import Client
+
+from .models     import Category, SubCategory, Product, ProductImage, Option, ProductAddress, ExpirationDate, Tag, ProductTag
+from users.models import User
+from reviews.models import Review
+
+class CategoryTest(TestCase):
+    def setUp(self):
+        Category.objects.create(
+            id = 1,
+            name = '아웃도어',
+            image_url = 'https://images.unsplash.com/photo-1501555088652-021faa106b9b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1473&q=80'
+        )
+
+        Category.objects.create(
+            id = 2,
+            name = '피트니스',
+            image_url = 'https://images.unsplash.com/photo-1579126038374-6064e9370f0f?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1431&q=80'
+        )
+
+        Category.objects.create(
+            id=3,
+            name = '공예DIY',
+            image_url = 'https://images.unsplash.com/photo-1488806374796-a4071c52353b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80'
+        )
```

Comment:
여기도 `bulk_insert`로 변경해주세요!
```suggestion
        Category.objects.create(
            id = 1,
            name = '아웃도어',
            image_url = 'https://images.unsplash.com/photo-1501555088652-021faa106b9b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1473&q=80'
        )

        Category.objects.create(
            id = 2,
            name = '피트니스',
            image_url = 'https://images.unsplash.com/photo-1579126038374-6064e9370f0f?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1431&q=80'
        )

        Category.objects.create(
            id=3,
            name = '공예DIY',
            image_url = 'https://images.unsplash.com/photo-1488806374796-a4071c52353b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1470&q=80'
        )
```